README.md

```diff
@@ -35,11 +35,12 @@ Apache Gravitino is a high-performance, geo-distributed, and federated metadata
 ![Gravitino Architecture](docs/assets/gravitino-architecture.png)
 
 Gravitino aims to provide several key features:
-
-* Single source of truth for multi-regional data with geo-distributed architecture support.
-* Unified data and AI asset management for both users and engines.
-* Security in one place, centralizing the security for different sources.
-* Built-in data management and data access management.
+* Unified Metadata Management: Gravitino provides a unified model and API to manage different types of metadata, including relational (e.g., Hive, MySQL) and file-based (e.g., HDFS, S3) metadata sources.
+* End-to-End Data Governance: Gravitino offers a unified governance layer for managing metadata with features like access control, auditing, and discovery.
+* Direct Metadata Management: Gravitino connects directly to metadata sources via connectors, ensuring changes are instantly reflected between Gravitino and the underlying systems.
+* Geo-Distribution Support: Gravitino enables deployment across multiple regions or clouds, allowing instances to share metadata for a global cross-region view.
+Multi-Engine Support: Gravitino supports query engines enabling metadata access without modifying SQL dialects.
```

Comment:
Do you need a "*" for "Multi-Engine Support"?

Reply:
Yes we do and done.